mp2-v1/tests/common/table.rs

```diff
@@ -560,8 +560,8 @@ impl Table {
         let zk_columns = self.columns.to_zkcolumns();
         Ok(ZkTable {
             // NOTE : we always look data in the row table
-            name: self.row_table_name(),
-            user_name: self.name.clone(),
+            zktable_name: self.row_table_name(),
```

Comment:
I would frankly call it `row_tree_table` even, since that's really the **only** one we're using no ?